Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -347,18 +347,17 @@ VideoConfigDiag::VideoConfigDiag(wxWindow* parent, const std::string &title, con
 
 	// Internal resolution
 	{
-	const wxString efbscale_choices[] = { _("Auto (Window Size)"), _("Auto (Multiple of 640x528)"),
-		_("Native (640x528)"), _("1.5x Native (960x792)"), _("2x Native (1280x1056) for 720p"), _("2.5x Native (1600x1320)"),
+	const wxString efbscale_choices[] = { _("Auto (Multiple of 640x528)"), _("Native (640x528)"), _("2x Native (1280x1056) for 720p"),
 		_("3x Native (1920x1584) for 1080p"), _("4x Native (2560x2112) for 1440p"), _("5x Native (3200x2640)"),
 		_("6x Native (3840x3168) for 4K"), _("7x Native (4480x3696)"), _("8x Native (5120x4224) for 5K"), _("Custom") };
 
 	wxChoice *const choice_efbscale = CreateChoice(page_enh,
-		vconfig.iEFBScale, wxGetTranslation(internal_res_desc), (vconfig.iEFBScale > 11) ?
+		vconfig.iEFBScale, wxGetTranslation(internal_res_desc), (vconfig.iEFBScale > SCALE_MAX_GUI) ?
```

Comment:
Do you need SCALE_MAX_GUI at all?

const int SCALE_MAX_GUI = sizeof(efbscale_choices) / sizeof(efbscale_choices[0]);